Laud_Misc_610/CGH/_1.trig

```diff
@@ -588,6 +610,20 @@
     irishRel:numChild 7;
     owl:sameAs <http://example.com/Rawl_B502/mínigud_na_cróeb_coibnesta_inso_sís.trig#FeideilmidRechtaid>.
 
+<#FeidilmidCrisAirgit>
+        a foaf:Person ;
+        irishRel:nomName "Feidilmid Cris Airgit" ;
+        irishRel:ancestorOfGroup <#DailFiatachEle>.
+
+<#DailFiatachEle>
+        a irishRel:PopulationGroup ;
+        irishRel:populationGroupName "Dail Fiatach Ele" .
+
+<#FeidlimidRechtmar>
+        a foaf:Person ;
+        irishRel:nomName "Feidlimid Rechtmar" ;
+        rel:childOf <#FeidilmidCrisAirgit>.
```

Comment:
> Feidilmid Cris Airgit a quo Dail Fiatach Ele & Feidlimid Rechtmar

While I can see how this note can be taken this way, read in isolation, I don't think it is trying to say that Feidlimid Rechtmar is a son of Feidlimid Cris Airgit. He's normally understood to be a son of Tuathal Techtmar. [Going back to Meyer's edition](https://archive.org/details/zeitschriftfrc08meyeuoft/page/292/mode/2up), this italicised passage in the CELT edition is apparently a variant reading from BB, which has for some reason been inserted into the CELT edition of Laud (Why do they do this sort of thing?!). So I don't think we should be encoding it here.